src/renderer/components/publication/PublicationCard.tsx

```diff
@@ -123,6 +124,19 @@ class PublicationCard extends React.Component<PublicationCardProps, PublicationC
         }
     }
 
+    private handleEnterPressed(e: React.KeyboardEvent) {
+        const {publication, lsdStatus} = this.props;
+        e.preventDefault();
+
+        if (e.key === "Enter") {
+            if (this.props.isOpds || !lcpReadable(publication, lsdStatus)) {
+                this.props.openInfosDialog(publication);
+            } else {
+                this.props.openReader(publication);
+            }
+        }
+    }
+
```

Comment:
Sorry for nit-picking, but this function duplicates pretty much the entire `handleBookClick()` body.

Can we instead filter the event type upstream of the function call? For example instead of:
`onKeyPress={(e) => this.handleEnterPressed(e)}`
...simply use:
`onKeyPress={(e) => { if (e.charCode === 13) { this.handleBookClick(e); } }}`

Note that I am suggesting using `e.charCode === 13` instead of `e.key === "Enter"`, in order to be consistent with other cases in the codebase.

PS: maybe replace `any` with `React.SyntheticEvent` in `handleBookClick(e: any)`? (this is just from the top of my head ... I haven't actually run the TypeScript compiler on this :)